runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowOperationContext.java

```diff
@@ -276,6 +276,23 @@ protected String getLullMessage(Thread trackedThread, Duration lullDuration) {
       return message.toString();
     }
 
+    protected String getBundleLullMessage(Thread trackedThread, Duration lullDuration) {
+      StringBuilder message = new StringBuilder();
+      message.append("Operation ongoing");
+      if (getStepName() != null) {
+        message.append(" in bundle ").append(getStepName().userName());
```

Comment:
ditto on step name being confusing for bundle and ditto about sharing code.

Reply:
Done. removed the step name